### DISCUSSION TITLE
Multiple fleet nodes

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#10530] [2.4] [answered]
Will it be possible to have multiple fleet nodes a deployment?

**Comment by defensivedepth** [comment, marked as answer]
Yes, you can have multiple Fleet nodes.

Would like to hear more about your use case - multiple Fleet nodes because of the amount of agents or something else?

**Reply by OzzyKampha** [reply]
The use case am having is both the amount of agents, but also agent crossing big geolocation, i dont want all 10k+ agents to call home, with multiple fleet servers i can separate on geoloc
